### ISSUE TITLE
Add `credential_subject` on `globus endpoint show`

### ISSUE THREAD

**Issue by sirosen** [globus/globus-cli#398]
Blocked on API support. This is just another Endpoint field. If it's added to the API, we can start displaying it.

The Hosted CLI has no details on this field in the manpage for `endpoint-details`.

I'm not confident this really has much value (other than internal debugging use cases), but if API support gets added I don't see any reason _not_ to add it to the CLI.

**Comment by ranantha** [comment]
Dan notes that he uses this for debugging, but users may not really look at this.

**Comment by sirosen** [comment]
Nobody has asked for this since CLI v1 released and it's been nearly a year since noted. Closing as a wontfix.
